.gitignore

```diff
@@ -64,6 +64,8 @@ out/
 
 # Builds may generate a CCache directory.
 ccache/
+ccache-downloads/
```

Comment:
Are these still used? I couldn't find anything outide `ccache/`.

Reply:
Not anymore - thx. I moved them inside the ccache folder itself. I'll remove.

Reply:
Removed.